java-security-test/src/test/resources/vcap.json

```diff
@@ -8,8 +8,8 @@
      "identityzone": "uaa",
      "identityzoneid": "uaa",
      "tenantmode": "dedicated",
-     "uaadomain": "auth.com",
-     "url": "https://paastenant.auth.com",
+     "uaadomain": "localhost",
```

Comment:
Hi Daniel,

i would remain this, because this checks other parts of the validation. Can't you just add another test-vcap-services.

Reply:
I have reversed those changes, added a new test and moved the integration tests into a new test class.